CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.8.4
```

Comment:
Just a heads-up, @spencer-codaio noticed we never released 0.8.2, so he has a PR out to consolidate all recent changes under 0.8.2. Whoever submits second will get a merge conflict.

Reply:
ah yeah @dweitzman-codaio mind rebasing and then adding this in under 0.8.2?

Reply:
I can then publish the npm version after you merge this one!

Reply:
Done